docs/basic-features/eslint.md

```diff
@@ -161,7 +149,19 @@ ESLint also contains code formatting rules, which can conflict with your existin
 
 ## Migrating Existing Config
 
-If you already have ESLint configured in your application, we recommend extending directly from the Next.js ESLint plugin instead of the shareable configuration.
+### Recommended Plugin Ruleset
+
+If you already have ESLint configured in your application and any of the following conditions are true:
+
+- You have one or more of the following plugins already installed (either separately or through a different config such as `airbnb` or `react-app`):
+  - `react`
+  - `react-hooks`
+  - `jsx-a11y`
+  - `import`
+- You've defined specific `parserOptions` that need to be different how Babel is configured within Next.js
+- You have `eslint-plugin-import` installed with Node and/or TypeScript resolvers defined
```

Comment:
```suggestion
- You have `eslint-plugin-import` installed with Node.js and/or TypeScript resolvers defined
```

Out of context, I don't understand what "resolvers" is - could be confusing for beginniners. Rephrase or add a link?

Reply:
Updated to include a little more detail as well as a link:

```diff
- You have `eslint-plugin-import` installed with Node.js and/or TypeScript resolvers defined
+ You have `eslint-plugin-import` installed with Node.js and/or TypeScript [resolvers](https://github.com/benmosher/eslint-plugin-import#resolvers) defined to handle imports
```